feature-libs/pickup-in-store/root/pickup-in-store-constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** The used for storing the name of the user's preferred store in browser local storage. */
+export const PREFERRED_STORE_LOCAL_STORAGE_KEY = 'preferred_store';
```

Comment:
should not put user's preferred store in localstorage. Leaking personal data can cause security issue.

Reply:
We had a team discussion and it was actually from our PO requirement, I am happy to set a call please let me know.